templates/fine-tune-llm_v2/training_configs/full_param/mistral-7b.yaml

```diff
@@ -1,20 +1,18 @@
-model_id: meta-llama/Llama-2-13b-chat-hf # <-- change this to the model you want to fine-tune
+model_id: mistralai/Mistral-7B-Instruct-v0.1 # <-- change this to the model you want to fine-tune
 train_path: s3://air-example-data/gsm8k/train.jsonl # <-- change this to the path to your training data
 valid_path: s3://air-example-data/gsm8k/test.jsonl # <-- change this to the path to your validation data. This is optional
-context_length: 4096 # <-- change this to the context length you want to use
+context_length: 512 # <-- change this to the context length you want to use
 num_devices: 16 # <-- change this to total number of GPUs that you want to use
 num_epochs: 10 # <-- change this to the number of epochs that you want to train for
 train_batch_size_per_device: 8
-eval_batch_size_per_device: 8
+eval_batch_size_per_device: 16
```

Comment:
We usually never deviate from train and eval batch size being the same, doing so just guranteed to not oom during evaluation, because you don't have backward pass.